.dockerignore

```diff
@@ -0,0 +1,22 @@
+# build/runtime artifacts from host OS
+node_modules/*
+tmp/*
+log/*
+vendor/cache*
+public/packs*
+docker
+
+# Dev cruft
+*.swp
+*.swo
+.sass-cache
+yarn-error.log
+.irb-history
+.pry_history
+.pryrc
```

Comment:
Why are we excluding these?  I find having a synchronized pry history especially useful.

Reply:
We can put them back in. I was excluding  cache, tmp, env-specific like things. Pry just got dragged in without too much thought